bestiary/core/schema.py

```diff
@@ -20,10 +20,176 @@
 #     Miguel Ángel Fernández <mafesan@bitergia.com>
 #
 
+import json
+
 import graphene
 import graphql_jwt
 
+from django.conf import settings
+from django.core.paginator import Paginator
+
+from django_mysql.models import JSONField
+
+from graphene.types.generic import GenericScalar
+
+from graphene_django.converter import convert_django_field
+from graphene_django.types import DjangoObjectType
+
 from .decorators import check_auth
+from .models import (Transaction,
+                     Operation)
+
+
+@convert_django_field.register(JSONField)
+def convert_json_field_to_generic_scalar(field, registry=None):
+    """Convert the content of a `JSONField` loading it as an object"""
+
+    return OperationArgsType(description=field.help_text, required=not field.null)
+
+
+class PaginationType(graphene.ObjectType):
+    page = graphene.Int()
+    page_size = graphene.Int()
+    num_pages = graphene.Int()
+    has_next = graphene.Boolean()
+    has_prev = graphene.Boolean()
+    start_index = graphene.Int()
+    end_index = graphene.Int()
+    total_results = graphene.Int()
+
+
+class OperationArgsType(GenericScalar):
+    @classmethod
+    def serialize(cls, value):
+        value = super().serialize(value)
+        value = json.loads(value)
+        return value
+
+
+class OperationType(DjangoObjectType):
+    class Meta:
+        model = Operation
+
+
+class TransactionType(DjangoObjectType):
+    class Meta:
+        model = Transaction
+
+
+class TransactionFilterType(graphene.InputObjectType):
+    tuid = graphene.String(required=False)
+    name = graphene.String(required=False)
+    is_closed = graphene.Boolean(required=False)
+    from_date = graphene.DateTime(required=False)
+    to_date = graphene.DateTime(required=False)
+    authored_by = graphene.String(required=False)
+
+
+class OperationFilterType(graphene.InputObjectType):
+    ouid = graphene.String(required=False)
+    op_type = graphene.String(required=False)
+    entity_type = graphene.String(required=False)
+    target = graphene.String(required=False)
+    from_date = graphene.DateTime(required=False)
+    to_date = graphene.DateTime(required=False)
+
+
+class AbstractPaginatedType(graphene.ObjectType):
+
```

Comment:
I think we should add documentation to the schema too.

Reply:
I have added this piece of documentation, but I was not sure about how detailed should I document the rest of the classes such as `PaginationType`.